src/client/post/EditPost/EditPost.js

```diff
@@ -137,21 +141,21 @@ class EditPost extends Component {
       const serverObject = await getObject(forecastObjectId);
       forecastObject = getClientWObj(serverObject, this.props.locale);
     }
-
-    let linkedList = [forecastObject, ...objEntities.map(entity => entity.data.object)];
-    if (!isRemovedCryptoObj) linkedList = [cryptoObject, ...linkedList];
-
-    return compact(uniqBy(linkedList, 'id'));
+    return compact(uniqBy([forecastObject, ...objEntities.map(entity => entity.data.object)], 'id'));
   };
 
   async handleChangeContent(rawContent) {
+    const { defaultObjects } = this.state;
     const nextState = { content: toMarkdown(rawContent) };
     this.getLinkedObjects(rawContent).then(linkedObjects => {
       const isLinkedObjectsChanged = !isEqual(this.state.linkedObjects, linkedObjects);
+      const linkedList = isLinkedObjectsChanged ? linkedObjects : this.state.linkedObjects;
+      const postObjects = !isEmpty(defaultObjects) ? [...defaultObjects, ...linkedList] : linkedList;
```

Comment:
you could omit isEmpty condition if you add 'defaultObjects' to init state

Reply:
++